server/server.go

```diff
@@ -178,7 +177,7 @@ func (server *HTTPServer) readReportForCluster(writer http.ResponseWriter, reque
 }
 
 // readRuleForReport return rule by cluster ID, org ID and rule ID
-func (server *HTTPServer) readRuleForReport(writer http.ResponseWriter, request *http.Request) {
+func (server *HTTPServer) singleRuleEndpoint(writer http.ResponseWriter, request *http.Request) {
```

Comment:
And this one, why was added `Endpoint`, if it's not used anywhere else?

Reply:
`readRuleForReport` is something weird. What does `for report` stand for? Maybe we could name it `readSingleRule` or something like this?

Reply:
Again, it can be subjective, I can revert your version, which I personally don't like.

Reply:
`readSingleRule` could be

Reply:
done